### DISCUSSION TITLE
Welcome to Plover Discussions!

### DISCUSSION THREAD

**Discussion by panathea** [opensteno/plover#1205] [General]
## 👋 Welcome! 🐦

  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Develop ideas using a more async and focused format than the Plover Discord allows
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

## Discord

Looking to chat with members of the community in a synchronous fashion? Check our the [Plover Discord community](https://discord.me/plover)

**Comment by petercpark** [comment]
Wow! this is very cool

**Comment by Register111** [comment]
Hi there,

  Anyone know how to restore the default settings in Plover? I accidentally deleted the dictionaries that were in there, and I can't seem to get them back. I even tried deleting the application to reinstall it. TIA
